src/pages/runewords/runewords.ts

```diff
@@ -89,6 +96,13 @@ export class Runewords {
         }
     }
 
+    @watch('exclusiveType')
+    exclusiveTypeChanged() {
+        if (this._debouncedSearchItem) {
```

Comment:
I think you have the convention + watch triggering on it

exclusiveType since your type is named this, by appending `Changed` to it you trigger the convention. 
You should have watch OR the convention, not both. I prefer the watch since I find it more explicit.